eventbrite.py

```diff
@@ -87,262 +89,279 @@
-    print(data)
+        return print(f"HTTP Error: {resp.status}")
     return data
 
-def get_event_by_id(http,eventbrite_api_token, event_id):
+
+def get_event_by_id(eventbrite_api_token, event_id):
     url = f"{EVENTBRITE_API_BASE_URL}events/{event_id}/"
     headers = {
         'Authorization': eventbrite_api_token
     }
 
-    # Make the GET request
-    resp = http.request('GET', url, headers=headers)
+    # Make the request
+    req = urllib.request.Request(url, headers=headers)
+
+     # Perform the GET request
+    with urllib.request.urlopen(req) as resp:
+        data = resp.read().decode('utf-8')
 
-    # Check the status code to ensure a successful request
     if resp.status != 200:
-        print(f"HTTP Error: {resp.status}")
-        
-    data = resp.data.decode('utf-8')
-    print(data)
+        return print(f"HTTP Error: {resp.status}")
+    
     return data
 
-def get_events_by_organization(http,eventbrite_api_token, organization_id):
+def get_events_by_organization(eventbrite_api_token, organization_id):
     url = f"{EVENTBRITE_API_BASE_URL}organizations/{organization_id}/events"
     headers = {
         'Authorization': eventbrite_api_token
     }
 
-    # Make the GET request
-    resp = http.request('GET', url, headers=headers)
+    # Make the request
+    req = urllib.request.Request(url, headers=headers)
 
-    # Check the status code to ensure a successful request
+     # Perform the GET request
+    with urllib.request.urlopen(req) as resp:
+        data = resp.read().decode('utf-8')
     if resp.status != 200:
-        print(f"HTTP Error: {resp.status}")
-    data = resp.data.decode('utf-8')
-    print(data)
+        return print(f"HTTP Error: {resp.status}")
+    return data
         
-def create_event(http, eventbrite_api_token, organization_id ,event_details):
+def create_event(eventbrite_api_token, organization_id ,event_details):
     url = f"{EVENTBRITE_API_BASE_URL}organizations/{organization_id}/events/"
     headers = {
     'Authorization': eventbrite_api_token,
     'Content-Type': 'application/json'
     }
+    event_details_bytes = event_details.encode('utf-8')
 
-    # Make the POST request
-    resp = http.request('POST', url, headers=headers, body=event_details)
+    req = urllib.request.Request(url, headers=headers, data=event_details_bytes, method='POST')
 
-    # Check the status code to ensure a successful request
-    if resp.status != 200:
-        error_data = json.loads(resp.data.decode('utf-8'))
-        error_detail = error_data.get('error_detail', {})
-        return {
-            "error": error_data.get('error', 'Unknown Error'),
-            "error_description": error_data.get('error_description', 'An error occurred'),
-            "error_detail": error_detail,
-            "status_code": resp.status
-        }
-    data = resp.data.decode('utf-8')
+    with urllib.request.urlopen(req) as resp:
+        data = resp.read().decode('utf-8')
+
+        if resp.status != 200:
```

Comment:
Dedent this, once you've read from resp, there's no need to stay within the context manager:
```py
with urllib.request.urlopen():
  data = resp.read().decode()

if resp.status ! = 200:
```